examples/Frameworker/Client/Program.cs

```diff
@@ -16,6 +16,7 @@
 
 #endregion
 
+using System.Net.Http;
```

Comment:
Why was this needed?

Reply:
I'm not sure. It seems like global namespaces for .NET 4.x were updated to remove System.Net.Http. I didn't investigate closely.